releases/v25_0_0.rst

```diff
@@ -1,17 +1,302 @@
+| Git tag                                   | 25.0.0     |
++-------------------------------------------+------------+
+| :doc:`EUPS distrib </install/newinstall>` | v25\_0\_0  |
++-------------------------------------------+------------+
+| rubin-env version                         | 5.0.1      |
++-------------------------------------------+------------+
+
+
+This release is based on the ``w_2022_48`` weekly build with 18 tickets backported.
+
+The notes below highlight significant technical changes to the Science Pipelines codebase in this release.
+For a complete list of changes made, see :doc:`tickets/v25_0_0`.
+
+The `Characterization Metric Report (DMTR-302) <https://dmtr-392.lsst.io/>`_ describes the scientific performance of this release in terms of scientific performance metrics.
+
+If you have questions or comments about this release, visit our `community forum <https://community.lsst.org>`_ for advice.
+
+- :ref:`release-v25-0-0-functionality`
+- :ref:`release-v25-0-0-interface`
+- :ref:`release-v25-0-0-pending-deprecations`
+- :ref:`release-v25-0-0-deprecations`
+
+*See also:*
+
+- :doc:`Installation instructions </install/index>`
+- :doc:`Known issues </known-issues>`
+- `Doxygen Documentation`__
+
+__ http://doxygen.lsst.codes/stack/doxygen/xlink_main_2022_11_24_08.00.57/
+
+.. _release-v25-0-0-functionality:
+
+Major New Features
+==================
+
+- :ref:`release-v25-0-0-analysis-tools`
+- :ref:`release-v25-0-0-gbdes`
+- :ref:`release-v25-0-0-parsl-plugin`
+- :ref:`release-v25-0-0-hips`
+
+.. _release-v25-0-0-analysis-tools:
+
+Analysis Tools
+--------------
+The new package, :ref:`analysis_tools <lsst.analysis.tools>`, implements a framework for modular, scalable, and reproducible computation of metrics and plots to assess both image and pipeline quality.
+
+.. _release-v25-0-0-gbdes:
+
+GBDES
+-----
+A significantly modified fork of Dark Energy Survey's astrometric calibration algorithm has been added to the Science Pipelines.
+`gbdes <https://github.com/lsst/gbdes>`_ will be used for global astrometric fitting in the data release pipeline from v26.0.0.
+See also :jira:`RFC-853`.
+
+.. _release-v25-0-0-parsl-plugin:
+
+BPS Parsl Plugin
+----------------
+A Parsl-based plugin `ctrl_bps_parsl <https://github.com/lsst/ctrl_bps_parsl>`_ for the LSST Batch Production Service (BPS) execution framework as added.
+It is intended to support running LSST PipelineTask jobs on high-performance computing (HPC) clusters.
+
+.. _release-v25-0-0-hips:
+
+Hierarchical Progressive Surveys (HiPS) trees
+---------------------------------------------
+
+`~lsst.pipe.tasks.hips.GenerateHipsTask` was added to the pipelines.
+This Task generates Hierarchical Progressive Surveys (HiPS) trees from high-resolution, healpix-projected coadds. HiPS images produced by this Task are viewable in the Portal for `Data Preview 0.2 <https://dp0-2.lsst.io/>`_.
+
+
+.. _release-v25-0-0-interface:
+
+Significant Interface Changes
```

Comment:
Remember we use sentence case for section headings per our Developer Guide and https://developers.google.com/style/highlights (look in all sections)